src/resources/runCommands.txt

```diff
@@ -1,14 +1,14 @@
-// Download spark
-wget https://archive.apache.org/dist/spark/spark-2.4.8/spark-2.4.8-bin-hadoop2.6.tgz
-
-// Increase driver memory
---driver-memory 8G
+// Download dependencies
+curl -OL https://downloads.datastax.com/dsbulk/dsbulk.tar.gz
+curl -OL https://downloads.datastax.com/enterprise/cqlsh-astra.tar.gz
+wget https://archive.apache.org/dist/spark/spark-2.4.8/spark-2.4.8-bin-hadoop2.7.tgz
 
 // Migrate
-spark-submit --properties-file /<path>/sparkConf.properties --verbose --master "local[8]" --class datastax.astra.migrate.Migrate /<path>/cassandra-data-migrator-1.x.jar
+spark-submit --properties-file /<path>/sparkConf.properties --verbose --master "local[8]" --class datastax.astra.migrate.Migrate /<path>/cassandra-data-migrator-2.x.jar
+spark-submit --properties-file /<path>/sparkConf.properties --master "local[8]" --driver-memory 25G --executor-memory 25G --class datastax.astra.migrate.Migrate /<path>/cassandra-data-migrator-2.x.jar &> table_out.log
```

Comment:
Is 25G here just an example? Did we already mention in the documentation that 32GB RAM or more is expected?

Reply:
Yes, this is just an example. Actual usage is based on use-case